server/CLI/manager.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/python3
+# Copyright 2019 Anton Maksimovich <antonio.maksimovich@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Utility for controlling (creating/deleting/printing) the clients instances. """
+
+import asyncio
+import re
+import os
+import secrets
+import ssl
+import string
+import sys
+from argparse import ArgumentParser
+from email.message import EmailMessage
+from smtplib import SMTP_SSL, SMTPException
+
+from asyncpg.exceptions import UniqueViolationError
+from dotenv import load_dotenv
+
+from huntflow_reloaded.models import User, gino_run  # pylint: disable=import-error
```

Comment:
We should remove pylint disabling at this line because "huntflow reloaded" will become a python package.
Every directory in project is a separate application which will being installed separately.
That is why we use `env PYTHONPATH=$(pwd)`.